__tests__/src/components/PackageListItem.test.js

```diff
@@ -2,43 +2,39 @@ import React from 'react';
 import { shallow } from 'enzyme';
 import expect from 'expect';
 
-import PackageListItem from '../../../src/components/PackageListItem';
+import PackageListItem from '../../../src/components/common/PackageListItem';
 
 describe("<PackageListItem />", () => {
   it("render one panel item", () => {
     const wrapper = shallow(<PackageListItem name="header"
-                                             packageUrl="core/demo-package" />);
-    expect(wrapper.find('div').length).toBe(4);
+                                             title="title"
+                                             publisherName="publisher"/>);
+    expect(wrapper.find('div').length).toBe(3);
   });
   it("panel header would be same as name property", () => {
     const panelHeader = "this is header";
     const wrapper = shallow(<PackageListItem name={panelHeader}
-                                             packageUrl="core/demo-package" />);
-    expect(wrapper.find(".panel-title").text()).toBe(panelHeader)
-  });
-  it("panel body would be same as description property", () => {
-    const panelHeader = "this is header";
-    const description = "this is panel description";
-    const wrapper = shallow(<PackageListItem name={panelHeader}
-                                             description={description}
-                                             packageUrl="core/demo-package" />);
-    expect(wrapper.find(".panel-body").text()).toBe(description);
+                                             title="title"
+                                             publisherName="publisher"/>);
+    expect(wrapper.find(".panel-title").text()).toBe('title [' + panelHeader + ']');
   });
   it("panel would have one button", () => {
```

Comment:
Again, we can avoid writing many small tests like this one and others, if we assert rendered html directly. One html test will be much easier to understand than many small ones.